ndc-lambda-sdk/src/connector.ts

```diff
@@ -6,11 +6,10 @@ import { RuntimeFunctions, executeMutation, executeQuery } from "./execution";
 
 export type Configuration = {
   functionsSchema: FunctionsSchema
+  runtimeFunctions: RuntimeFunctions
 };
```

Comment:
Why does this move out of state?

Reply:
Can you pass in functions in config, or conversely, export them?

Reply:
Ah yeah that makes sense. I guess the name still throws me off but the implementation is obviously fine.

Reply:
Yeah, I agree. I think the `State` and `Configuration` style of doing things is kinda a holdover from the old configuration server. In practice now, we really could combine `State` and `Configuration` in the SDK and let the connector authors draw a distinction inside the one type if they want. c'est la vie.